prisma/migrations/20220918154615_make_contributions_many_to_many/migration.sql

```diff
@@ -0,0 +1,82 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `contributor_id` on the `companies` table. All the data in the column will be lost.
+  - You are about to drop the column `contributor_id` on the `roles` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[company_id,title,type,year]` on the table `roles` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropForeignKey
+ALTER TABLE "companies" DROP CONSTRAINT "companies_contributor_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "roles" DROP CONSTRAINT "roles_contributor_id_fkey";
+
+-- AlterTable
+ALTER TABLE "companies" DROP COLUMN "contributor_id";
+
+-- AlterTable
+ALTER TABLE "roles" DROP COLUMN "contributor_id";
+
+-- CreateTable
+CREATE TABLE "companies_contributions" (
```

Comment:
Is the table suppose to be 
```suggestion
CREATE TABLE "company_contributions" (
```

Same for `role_contributions`?